src/form/Form.tsx

```diff
@@ -953,6 +962,63 @@ function Form({
     });
     if (invalid) return;
 
+    // Perform any Stripe payment method setup here if and only if the
+    // payment method field has a complete value
+    const paymentMethodFields = getPaymentMethodFieldsOnStep(activeStep);
+    const haveCompletePaymentMethod = paymentMethodFields.some((pmf: {servar: {key: string}}) =>
+      // @ts-expect-error
```

Comment:
can we add a comment to each expect-error so we know why we're leaving an error in?

Reply:
With other changes I made, much of this code moved out of Form.tsx.  I commented the remaining expected TS error.